documentation/modules/payload/reverse_ord_tcp.md

```diff
@@ -0,0 +1,189 @@
+windows/shell/reverse_ord_tmp:Windows Command Shell, Reverse Ordinal TCP Stager is an unique windows payload for Metasploit Framework.
+It is really small (<100 bytes), it uses the existing ws2_32.dll in memory in connect and load the next stage of the payload.
+
+It provides a shell on the target machine which can be used to achieve almost anything on the target pc.
+
+## Vulnerable Application
+
+This Meterpreter payload is suitable for the following environments:
+
+* Windows x64
```

Comment:
Trying to think what the first x64 win was... XP? Be good to day if it works back that far (assuming that's tested and easy to look up)

Reply:
Yeah, it works on x64 Win XP SP2 distro🙂